### PR TITLE
added vault/unvault docs

### PR DESCRIPTION
**Pull request by bcoca**
cause filters are still manual

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

doc filters

### CODE REVIEW

File: docs/docsite/rst/user_guide/playbooks_filters.rst [hunk 1]
```diff
@@ -1349,6 +1349,30 @@ Hash type 'blowfish' (BCrypt) provides the facility to specify the version of th
 
 .. versionadded:: 2.12
 
+You can also use Ansible Vault compatible filter to encrypt data::
```
Comment (acozine) [inline docs/docsite/rst/user_guide/playbooks_filters.rst]: Wordsmithing and adding a link here. We should definitely link to the more comprehensive docs on using vault, if we're going to do this.
```suggestion
You can also use the Ansible :ref:`vault <vault>` filter to encrypt data::
```

File: docs/docsite/rst/user_guide/playbooks_filters.rst [hunk 2]
```diff
@@ -1349,6 +1349,30 @@ Hash type 'blowfish' (BCrypt) provides the facility to specify the version of th
...
+    myvaultedkey: "{{ keyrawdata|vault(passphrase) }} "
+
+  - name: save templated vaulted data
+    template: src=dump_template_data.j2 dest=/some/key/vault.txt
+    vars:
+      mysalt: '{{2**256|random(seed=inventory_hostname)}}'
+      template_data: '{{ secretdata|vault(vaultsecret, salt=mysalt) }}'
+
+
+And then decrypt it via unvault::
```
Comment (acozine) [inline docs/docsite/rst/user_guide/playbooks_filters.rst]: Remove the latin:
```suggestion
And then decrypt it using the unvault filter::
```

**Review by acozine — COMMENTED** [review]
Is this PR in response to a user issue? The filter docs aren't the best place to discuss the risks of encrypting things with vault. If there's a compelling reason to add this, then let's clean up the writing a bit and link to the more comprehensive docs on vault.

**Comment by bcoca** [comment]
@acozine this i hope is just temporary until we can use these https://github.com/ansible/ansible/blob/245a2a85c4c7073afcb677fdf737aec2f7cca11c/lib/ansible/plugins/filter/vault.yml

**Review by samccann — APPROVED** [review]

**Pull request merged**